packages/nuxt-locokit/src/stores/users.ts

```diff
@@ -1,175 +1,57 @@
-      return res
-    } catch (err) {
-      // eslint-disable-next-line no-console
-      console.error(err)
-      error.value = err as Error
-    }
-    loading.value = false
-  }
-
-  async function findUsers(
-    params: Record<string, any>,
+  async function updateUsers(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    params?: Record<string, any>,
     sort = {
       createdAt: -1,
     },
   ) {
     loading.value = true
     error.value = null
-    try {
-      users.value = await sdkClient.service('user').find({
-        query: {
-          params,
-          // $sort: sort,
-        },
-      })
-    } catch (err) {
-      // eslint-disable-next-line no-console
-      console.error(err)
-      error.value = err as Error
-    }
-    loading.value = false
-  }
-
-  async function searchUsers({
-    query,
-    filters,
-    pageIndex = 0,
-    limit = ITEMS_PER_PAGE,
-    sort = {
-      createdAt: -1,
-    },
-  }: {
-    query?: string | null
-    filters?: Filter[] | null
-    pageIndex?: number
-    limit?: number
-    sort?: Record<string, number> | null
-  }) {
-    loading.value = true
-    error.value = null
-    try {
-      let result = null
-      if (filters && query) {
-        result = await sdkClient.service('user').find({
-          query: {
-            username: {
-              $ilike: `%${query}%`,
-            },
-            ...getCurrentFilters(filters),
-            $limit: limit,
-            $skip: pageIndex * limit,
-            // $sort: sort,
-          },
-        })
-      } else if (query) {
-        result = await sdkClient.service('user').find({
-          query: {
-            username: {
-              $ilike: `%${query}%`,
-            },
-            $limit: limit,
-            $skip: pageIndex * limit,
-            // $sort: sort,
-          },
-        })
-      } else if (filters) {
-        result = await sdkClient.service('user').find({
-          query: {
-            ...getCurrentFilters(filters),
-            $limit: limit,
-            $skip: pageIndex * limit,
-            // $sort: sort,
-          },
-        })
-      }
-      loading.value = false
-      return result
-    } catch (err) {
-      // eslint-disable-next-line no-console
-      console.error(err)
-      error.value = err as Error
+    const res = await findUsers(params, sort)
+    if (res instanceof Error) {
+      error.value = res
+    } else {
+      users.value = res
     }
     loading.value = false
   }
 
   async function blockAccountUser(id: string, blocked: boolean) {
     loading.value = true
     error.value = null
-    try {
-      const res = await sdkClient.service('user').patch(id, {
-        blocked: !blocked,
-      })
-      loading.value = false
-      return res
-    } catch (err) {
-      // eslint-disable-next-line no-console
-      console.error(err)
-      error.value = err as Error
+    const res = await patchUser(id, {
+      blocked: !blocked,
```

Comment:
If the aim of the function is to `block` the account user, I think we have to set `blocked` to `true`.

Or maybe, this is a "toggle" function, so prefer name it `toggleBlockAccountUser` or something else, telling the developer the function will toggle the value of `blocked`.